content/apps/managed-services.md

```diff
@@ -25,54 +21,42 @@ The v1 services in [cf-services-contrib-release](https://github.com/cloudfoundry
 
 	brew install jq
```

Comment:
yes I just copy/pasted that from the old doc. I will create new PR that points at setup page.